app/models/vote.rb

```diff
@@ -0,0 +1,17 @@
+class Vote < ApplicationRecord
+  belongs_to :work
+  belongs_to :user
+
+  validates :work_id, presence: true
```

Comment:
You should also have a validation that the [unique combination](https://stackoverflow.com/questions/34424154/rails-validate-uniqueness-of-two-columns-together) of vote_id and work_id are unique.